src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2019_03_01/test_vm_commands.py

```diff
@@ -1831,7 +1831,7 @@ def test_vmss_create_linux_secrets(self, resource_group):
             'loc': 'westus',
             'vmss': 'vmss1-name',
             'secrets': json.dumps([{'sourceVault': {'id': 'id'}, 'vaultCertificates': []}]),
-            'vault': self.create_random_name('vmlinuxkv', 20),
```

Comment:
It's randomized. Why duplicate?

Reply:
It's fake randomized, just increase by one each time it's called.